ang/civicase/CaseListTable.html

```diff
@@ -7,16 +8,16 @@
         <thead class="civicase__case-list-table__header">
         <tr>
           <th class="civicase__case-list-column--fixed civicase__case-list-column--first">
-            <span class="case-list-checkbox" ng-if="!viewingCase && selectedCases && bulkAllowed && !item.lock">
-              <input type="checkbox" ng-click="selectAll($event)" id="select-role-{{ $index }}" ng-checked="isSelection('all')" />
-              <label for="select-role-{{ $index }}"></label>
-            </span>
-            <span class="btn-group btn-group-sm" ng-if="!viewingCase && selectedCases && bulkAllowed && !item.lock">
-              <button type="button" class="btn btn-default dropdown-toggle" ng-class="{disabled: !selectedCases.length}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                {{ ts('Actions') }}
-                <span class="caret"></span>
-              </button>
-              <ul class="dropdown-menu" civicase-actions="selectedCases" multiple="true" refresh-callback="refresh"></ul>
+            <!--  -->
+            <civicase-bulk-actions-checkboxes ng-show="!viewingCase && selectedCases && bulkAllowed && !item.lock" class="pull-left" show-checkboxes="showCheckboxes" is-select-all-available="isSelectAllAvailable" selected-items="selectedCases.length"></civicase-bulk-actions-checkboxes>
+            <span class="civicase__bulkactions-actions btn-group btn-group-sm" ng-if="!viewingCase && selectedCases && bulkAllowed && !item.lock">
```

Comment:
Same as the other comment
> We should need to mention this many conditions.
$parent.showCheckboxes && !$parent.viewingCase && $parent.selectedCases && $parent.bulkAllowed

> When bulkAllowed is false, showCheckboxes should be disabled by default. In fact the Bulk Allowed logic should be part of BulkActions directive. And unless user have permission, showCheckboxes should be disabled by default.

> Also could you remind me, why !$parent.viewingCase && $parent.selectedCases is needed to be checked here?

Reply:
Please see my comments above